interview-questions.md

```diff
@@ -7,62 +7,73 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statement is checking an input is one or the other. It is using if/else-if/else statements to verify a condition. 
+  
+  An example is if the number 7 is an even or odd number. Checking if a number is even or odd is a conditional statement and using variable inputs of numbers to check it.
 
-  Researched answer:
+  Researched answer: Conditional statements are used in all programming languages to let a computer know if a condition is met and how to execute them. Generally, condtional statements are used with if/else-if/else, but there are other ways to execute them as well. Examples: elif (Python) and switch cases (C#, Java, JavaScript)
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is a version control system that lives on individual computers and used to track what version of code you are working on. GitHub is a software that lives on the cloud. It is used to upload different branches of code to collaborate with anyone around the world. The main difference between the two is that they are 2 separate systems and used for 2 separate things. However, they are 2 systems/software that work together in order to achieve one goal: Maintaining 1 main branch that everyone can work off of and ensure everyone is on the same page.
 
-  Researched answer:
+  Researched answer: Git is a version control system. GitHub is a cloud-based hosing service. Git is installed locally on a system. GitHub lives in the cloud so internet is required.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: JavaScript operators that will return a boolean value (true, false) would be >, <, <=, >=, &&, ||.
```

Comment:
While correct, you should be expanding on this a lot more.  If asked this in an interview, you should be talking about each one of those operators and what their intent is.  One-line answers here are considered the bare minimum for the requirements of this section and will come off as uninterested in an interview.